README.md

```diff
@@ -9,39 +9,50 @@ This repository contains the backend API for the Giraf Project. The API is a .ne
 To run the project locally with a MySQL database first do the following:
 
 0. Prerequisites:
-  - Download and install .NET Core 2.1.101 SDK or a version backwardly compatible with it.
-  - Download and install MySQL Server and optionally MySQL Workbench or another database manager.
-  - Create a database named giraf on the MySQL Server. This can be done from the workbench or via cli.
-
-1. Open a terminal-emulator and navigate to `⋯/web-api/GirafRest`
-
-  - run `cp appsettings.template.json appsettings.Development.json`
-  - update appsettings.Development.json with
-	- All fields <> in the"DefaultConnection" string.
-    - Jwt.JwtKey field, which must be any (random) string of, at least 40, alpha-numeric charecters
-    - Jwt.JwtIssuer field, which is your name or organization. For example "Aalborg University"
-  - run `dotnet restore`
-  - run `dotnet ef database update`
-  - run `dotnet run --sample-data`
-
-The `--sample-data` flag will generate sample data if target database is empty, and we recommend using this flag by default. Other possible flags are:
+  - Download and install .NET Core 2.2 SDK or a version backwardly compatible with it.
+  - Download MySQL installer
+  	* Install MySQL server 5.7, under the setup, create a root account with password “password”, and add a user with username “user” with password “password”.
+	* (Optional) Install Workbench
+  - Setup a giraf database/schema named giraf on the MySQL Server (can be done from Workbench or via cli).
+    * (Through Workbench) Start MySQL Workbench. There should be a Local instance running under MySQL Connections, log in using the created root password, “password”, create a new schema named giraf
```

Comment:
Also give an explanation for how to do it through the shell. As you say the workbench is optional.

Reply:
Added cli guide